resources/kernel/make_kernel_image_deb.sh

```diff
@@ -103,11 +109,19 @@ if [ -n "$MAKEDEB" ]; then
     cp "$kernel_dir"/../*.deb "$kernel_dir/deb"
 else
     echo "Downloading deb packages..."
+    if [[ $arch = "x86_64" ]]; then
```

Comment:
This if/elseif is missing formatting. The code inside the blocks should be aligned with one tab more to the right.